src/_P020_Ser2Net.ino

```diff
@@ -18,6 +18,20 @@ byte    Plugin_020_SerialProcessing = 0;
 WiFiServer *ser2netServer;
 WiFiClient  ser2netClient;
 
+void P20_sendevent(int val)
+{
+  if (Settings.UseRules)
+  {
+    String RuleEvent;
+    RuleEvent += getTaskDeviceName(event->TaskIndex);
```

Comment:
Where is this `event` coming from?
It isn't a parameter of the function.

Reply:
right, but surprise is why it compiles ;)

Reply:
[It doesn't](https://github.com/letscontrolit/ESPEasy/actions/runs/761583229)

Reply:
ok now will be better, but this link shows all as success

Reply:
> ok now will be better, but this link shows all as success

Yep, the script isn't checking well for the result. Still have to look into it.